File: python/bc1117.py
i = 0
s = 0.0

while i < 2:
    val = float(input())

    if val < 0.0 or val > 10.0: print("nota invalida")
    else:
        i += 1
        s += val

print(f"media = {s / 2:.2f}")

File: cpp/bc1117.cpp
#include <iostream>
#include <iomanip>

using namespace std;

int main()
{
	int i = 0;
	float val = 0.0, s;
	
	while(i != 2)
	{
		cin >> val;
		
		if(val >= 0.0 && val <= 10.0)
		{
			s += val;
			i++;
		}
		else cout << "nota invalida" << endl;
	}
	
	cout << "media = " << fixed << setprecision(2) << s / 2.0 << endl;
}
